db/migrate/20211207124418_make_space_group_optional.rb

```diff
@@ -0,0 +1,7 @@
+class MakeSpaceGroupOptional < ActiveRecord::Migration[6.1]
+  def change
+    change_column_null :spaces, :space_group_id, true
+    # Remove any empty SpaceGroups set by mistake earlier:
+    SpaceGroup.where(title: "").destroy_all
```

Comment:
We should avoid using ActiveRecord models in migrations and stick to pure SQL, this is because the model may change in the future and this migration would stop working

`execute("DELETE FROM space_groups WHERE space_groups.title = "");`

Reply:
Takk!

Reply:
@mathias234 Denne kodesnutten gjør også jobben med å sette alle space_group_id til "nil" på Spaces og SpaceContacts som er assosiert med SpaceGroup som hadde title: "". 

Det er høyst usannsynlig at dette vil være et problem for oss i fremtiden, da det er vel ingen andre enn oss som kommer til å kjøre en versjon før SpaceGroup ble optional. Skal vi bare lage en engangs-seed for det, og kjøre den - i stedet for å ta det ut av ActiveRecord eller lage SQL queries for hele greia?

Reply:
Hm ja er sikkert greit det

Reply:
@mathias234 Fixed now